.gitignore

```diff
@@ -9,6 +9,8 @@
 !.yarn/plugins
 !.yarn/releases
 !.yarn/versions
+package.json
+package-lock.json
```

Comment:
please undo these changes, they are not needed at all

Reply:
here we go!